pkg/prometheus_exporter/exemplars.go

```diff
@@ -0,0 +1,85 @@
+package prometheus_exporter
+
+import (
+	"fmt"
+	"github.com/golang/protobuf/proto"
+	"github.com/golang/protobuf/ptypes"
+	"github.com/prometheus/client_golang/prometheus"
+	dto "github.com/prometheus/client_model/go"
+	"time"
+	"unicode/utf8"
+)
+
+// Support for exemplars is still considered experimental both, in Prometheus and in the client library.
```

Comment:
Shouldn't we mark the exemplars support in slo-exporter as experimental as well?

I would rather prefer to do so.

Reply:
Fair enough, I'll think of some reasonable way where to add the note about it

Reply:
added here a9fd1b57a177974e1880ee78785215e857adc099